canonical/charmhub.io
# Move search to homepage hero

## Done
Removed the search from the navigation and changed the hero to include the search

## How to QA
- Go to https://charmhub-io-1692.demos.haus/
- Check that there is no search in the navigation
- Check that the hero contains the search
- Check that the search works as expected

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-6773
